docs/api/cluster/icdm.rst

```diff
@@ -24,10 +32,29 @@ Intercluster distance maps display an embedding of the cluster centers in 2 dime
     visualizer.fit(X)        # Fit the data to the visualizer
     visualizer.show()        # Finalize and render the figure
 
+
+Quick Method
+------------
+
+The same functionality above can be achieved with the associated quick method `silhouette_visualizer`. This method will build the Silhouette Visualizer object with the associated arguments, fit it, then (optionally) immediately show it.
+
+.. plot::
+    :context: close-figs
+    :alt: ICDM Quick Method
+
+    from yellowbrick.datasets import load_nfl
+    from sklearn.cluster import MiniBatchKMeans
+    from yellowbrick.cluster import intercluster_distance
+
+
+    X, y = load_nfl()
```

Comment:
Not sure if the documentation build complains "unused variable y" - but since we have to make a change to this document anyway, perhaps we should make this `X, _ = load_nfl()`?